modules/objdetect/src/qrcode.hpp

```diff
@@ -0,0 +1,56 @@
+// This file is part of OpenCV project.
```

Comment:
remove this file from the patch. Put the class declaration right into the .cpp file. Put the external function declaration to the objdetect.hpp file.